src/org/omegat/core/KnownException.java

```diff
@@ -47,7 +49,7 @@ public KnownException(Throwable ex, String errorCode, Object... params) {
     }
 
     public Object[] getParams() {
-        return params;
+        return Arrays.copyOf(params, params.length, Object[].class);
```

Comment:
This can be 
```suggestion
        return Arrays.copyOf(params, params.length);
```

Reply:
There is no usage of parameters for creation of the `KnownException` in the OmegaT source code.
It may be better to remove a field params and update constructor without params.